operator/cluster/upgrade.go

```diff
@@ -1,7 +1,7 @@
 package cluster
 
 /*
- Copyright 2017 - 2020 Crunchy Data Solutions, Inc.
+ Copyright 2019 Crunchy Data Solutions, Inc.
```

Comment:
Please revert this back to `2017 - 2020`

Reply:
updated.